### DISCUSSION TITLE
How to delay cross tab messages?

### DISCUSSION THREAD

**Discussion by aman-godara-ria** [SupremeTechnopriest/react-idle-timer#383] [Q&A]
I want to delay user actions performed on one tab to being messaged to other tabs. If needed I can explain the reason why I am asking for this. Is there anything in react-idle-timer that can help me with this? 

`syncTimer` feature doesn't delay the messages, its purpose is to throttle them which helps to reduce the amount of messages going through.

**Comment by aman-godara-ria** [comment]
I instead put a delay inside the function that I was passing to `onMessage` prop.

**Comment by SupremeTechnopriest** [comment]
Thats a good solution 👍. I think this is out of scope for IdleTimer core. Glad you figured it out.  Going to close this.
